docs/user-guide/mvd-editor.md

```diff
@@ -2,40 +2,71 @@
 
 With the Zowe Editor, you can create and edit the many types of files.
 
-## Specifying a language server
+## Specifying a highlighting language
 
-To specify a language server, complete these steps:
+1. Click **Language** on the editor toolbar. A dropdown menu will be displayed.
```

Comment:
In some areas you use "editor toolbar" but in some other areas we still have "from the *something* menu" we should use the same term for that component in the Editor. In areas of the code interally, it's referred to as the "Menu bar" so we can probably call it that, or Toolbar whichever you prefer/is more formal

Reply:
Things like the "right-click menu" are good, because that's an actual context menu & not referring to the menu bar

Reply:
Also in Hotkeys, we have "Menu bar" 😃